src/vma/main.cpp

```diff
@@ -269,16 +269,20 @@ void check_debug()
 void check_flow_steering_log_num_mgm_entry_size()
 {
 	char flow_steering_val[2] = {0};
-	if (priv_read_file((const char*)FLOW_STEERING_MGM_ENTRY_SIZE_PARAM_FILE, flow_steering_val, 2) == -1) {
+	if (priv_read_file((const char*)FLOW_STEERING_MGM_ENTRY_SIZE_PARAM_FILE, flow_steering_val, 2, VLOG_DEBUG) == -1) {
 		vlog_printf(VLOG_DEBUG, "Flow steering option does not exist in current OFED version");
 	}
 	else if (flow_steering_val[0] != '-' || flow_steering_val[1] != '1') {
 		vlog_printf(VLOG_WARNING, "***************************************************************************************\n");
-		vlog_printf(VLOG_WARNING, "* VMA will not operate properly while flow steering option is disabled!               *\n");
-		vlog_printf(VLOG_WARNING, "* Please restart your VMA applications after running the following:                   *\n");
-		vlog_printf(VLOG_WARNING, "* WARNING: the following steps will restart your network interface!                   *\n");
+		vlog_printf(VLOG_WARNING, "* VMA will not operate properly while flow steering option is disabled                *\n");
+		vlog_printf(VLOG_WARNING, "* In order to enable flow steering please restart your VMA applications after running *\n");
+		vlog_printf(VLOG_WARNING, "* the following:                                                                      *\n");
+		vlog_printf(VLOG_WARNING, "* For your information the following steps will restart your network interface        *\n");
 		vlog_printf(VLOG_WARNING, "* 1. \"echo options mlx4_core log_num_mgm_entry_size=-1 > /etc/modprobe.d/mlnx.conf\" *\n");
-		vlog_printf(VLOG_WARNING, "* 2. \"/etc/init.d/openibd restart\"                                                  *\n");
+		vlog_printf(VLOG_WARNING, "* 2. \"modprobe -r mlx4_ib\"                                                          *\n");
+		vlog_printf(VLOG_WARNING, "* 3. \"modprobe -r mlx4_en\"                                                          *\n");
+		vlog_printf(VLOG_WARNING, "* 4. \"modprobe -r mlx4_core\"                                                        *\n");
+		vlog_printf(VLOG_WARNING, "* 5. \"modprobe mlx4_core\"                                                           *\n");
```

Comment:
@rosenbaumalex README.txt should be revised as it is only pertains to VMA prior to RHEL 7.3. I suggest adding an updated README.txt as a follow up patch after technical writing review.
Need to add a new section for VMA since RHEL 7.3